src/hooks/useHasAppUpdated/useHasAppUpdated.test.ts

```diff
@@ -0,0 +1,80 @@
+import { renderHook } from '@testing-library/react-hooks'
+import { act } from '@testing-library/react-hooks'
+import axios from 'axios'
+
+import { useHasAppUpdated } from './useHasAppUpdated'
+import { APP_UPDATE_CHECK_INTERVAL } from './useHasAppUpdated'
+
+jest.mock('axios')
+const mockAxios = axios as jest.Mocked<typeof axios>
+
+global.window = Object.create(window)
+const url = 'http://dummy.com'
+Object.defineProperty(window, 'location', {
+  value: {
+    hostname: url
+  },
+  writable: true
+})
```

Comment:
this is going to pollute the jest test suite - please do this in beforeAll and revert it in afterAll